packages/core/framework/src/http/express-loader.ts

```diff
@@ -50,7 +51,17 @@ export async function expressLoader({ app }: { app: Express }): Promise<{
 
   let redisClient: Redis
 
-  if (configModule?.projectConfig?.redisUrl) {
+  if (configModule?.projectConfig.sessionOptions?.dynamodbOptions) {
```

Comment:
Q: Do we need to update the shutdown function or is it not necessary?

Reply:
When I looked into their source code, I found they do not have any long-lived connections like Redis. However, there is a `destroy` method to close the connection of an HTTP request, which seems like an HTTP request made by an executed DynamoDB command.

Reply:
![CleanShot 2025-04-16 at 14 52 32@2x](https://github.com/user-attachments/assets/039735cb-b30b-4bfa-9a9c-1096728e863c)

Calling `dynamoDbClient.destroy` will call this method under the hood and I think we don't want to do this.